vembrane/modules/annotate.py

```diff
@@ -67,8 +70,8 @@ def annotate_vcf(
 
     available_chromsomes = set(np.unique(ann_data[config_chrom_column]))
 
-    tree = dict()
-    chrom_ann_data = dict()
+    tree = {}
+    chrom_ann_data = {}
```

Comment:
I guess this was an automatic change by ruff, right? Just thinking that `dict()` is clear, while `{}` can theoretically be syntax for a `dict` or a `set`, right?

Reply:
I think `{}` is always a dict, but I'm not sure. In any case, we may just as well add the corresponding type annotations, i.e. `tree: dict[foo, bar] = {}`.

Reply:
Yeah, `{}` is always an empty `dict()`, but once you put something in there, it can be a set, e.g. `{'a'}`. So the syntax is clearly defined, I was just thinking it can be confusing to human code readers. So type annotations probably don't hurt. But also, this is a very minor concern...

Reply:
I added type annotations here (and also at some other places), so that it's clear we're dealing with dicts.